src/main/jbake/content/docs/what.adoc

```diff
@@ -0,0 +1,208 @@
+= Why Hawkular?
+Heiko Rupp
+2015-07-04
+:description: Vision for Hawkular
+:jbake-type: page
+:jbake-status: draft
+
+
+== Why Hawkular?
+
+.NOTE: WHY: what is the purpose, why Hawkular
+
+To be the best tool to monitor JBoss and other Java Middleware helping
+you spending less and more enjoyable time on middleware management.
+
+OR:
+
+Hawkular will reduce the effort and time spent on systems management and monitoring
+and make this task more enjoyable.
```

Comment:
We will need basic OS-level monitoring. Otherwise a lot of the advanced features will not work. Take CPU usage. If your app server is slow, you may want to see that another process is taking 99% cpu.